cmd/influx/main.go

```diff
@@ -83,6 +83,9 @@ func influxCmd() *cobra.Command {
 			cmd.Usage()
 		},
 	}
+
+	viper.SetEnvPrefix("INFLUX")
```

Comment:
caught this when testing the cmd after these changes - we should've been calling this before adding any of the commands below.